docs/testplan/ACL-Outer-Vlan-test-plan.md

```diff
@@ -0,0 +1,123 @@
+# ACL Outer Vlan_ID Test Plan
+
+- [Overview](#overview)
+  - [Scope](#scope)
+  - [Testbed](#testbed)
+- [Setup configuration](#Setup-configuration)
+- [Test Cases](#Test-cases)
+  - [test_egress_vlan_outer_forward](#Test-case-test_egress_vlan_outer_forward)
+  - [test_ingress_vlan_outer_forward](#Test-case-test_ingress_vlan_outer_forward)
+  - [test_ingress_vlan_outer_drop](#Test-case-test_ingress_vlan_outer_drop)
+  - [test_egress_vlan_outer_drop](#Test-case-test_egress_vlan_outer_drop)
+
+## Overview
+
+The purpose is to test the functionality of ACL feature on the SONiC switch DUT. The tests expecting that all necessary configuration for ACL are pre-configured on SONiC switch before test runs.
+
+## Scope
+
+The test is targeting a running SONiC system with fully functioning configuration.
+Purpose of the test is to verify a SONiC switch system correctly performs ACL actions based on match with Outer Vlan Id.
```

Comment:
>Purpose of the test [](http://example.com/codeflow?start=0&length=19)

By default the ports in vlan are tagged port In the test scope, could you test both tagged and untagged ports?

It will be even more complex if ports belong to multiple vlans.

There is a test gap [https://github.com/Azure/sonic-mgmt/issues/3623](https://github.com/Azure/sonic-mgmt/issues/3623)